controller.js

```diff
@@ -22,7 +22,20 @@ module.exports.getReviewById = (request, response, next) => {
             return Promise.reject({status: 404, msg: 'Review Id not found'})
         }
     }).catch(next)
-    
+}
+
+module.exports.getComByReviewId = (request, response, next) => {
+    let reviewId = request.params.review_id
+    fetchReviewById(reviewId).then(({rows, rowCount}) => {
+        if (rowCount === 0) {
```

Comment:
I'd move your promise.rejects into your models instead of having them in the controllers.